doc/source/migrations/index.rst

```diff
@@ -25,6 +25,23 @@ Development version
 
 - To set up your repository to use the ``ansys/actions/doc-changelog`` action, see the :ref:`docs_changelog_action_setup`.
 
+Version ``v6``
+--------------
+
+**New features:**
+- SEO improvements. These are implemented inside the `doc-deploy-dev
+  <https://actions.docs.ansys.com/version/stable/doc-actions/index.html#doc-deploy-dev-action>`_
+  and the `doc-deploy-stable
+  <https://actions.docs.ansys.com/version/stable/doc-actions/index.html#doc-deploy-stable-action>`_.
```

Comment:
@jorgepiloto - please add this to the "Development version" section. Once we release, we will change the title

Reply:
Sure! Documenting it there.